packages/execution-environments/jest.config.js

```diff
@@ -6,10 +6,10 @@ module.exports = {
   coverageReporters: ['text', 'html'],
   coverageThreshold: {
     global: {
-      branches: 10,
-      functions: 13,
-      lines: 10,
-      statements: 11,
+      branches: 27,
```

Comment:
Do we have a standard for this? I see it changed once in a while 🤔

Reply:
Not a "standard" per se, but we're trying to maximize coverage metrics, and my tests in this PR ended up increasing coverage, so I just bumped the minimums to match.